rdm/0006-profiles-and-flavours.md

```diff
@@ -0,0 +1,120 @@
+- Start Date: 2019-10-02
+- RFC PR: [#6](https://github.com/inveniosoftware/rfcs/pull/6)
+- Authors: Guillaume Viger
+
+# Customize created Invenio instance by choosing profile and optionally flavour
+
+## Summary
+
+When running the instance bootstrapping CLI command (`invenio-cli init`), the
+hoster should be given a choice to create an Integrated Library System (ILS) or
+a Resource Data Management (RDM) system among other Invenio application **flavours**. If RDM
+is chosen, there are further **profiles** to be chosen from, for instance "default"
+and "biomed". These choices pre-configure the application to varying levels.
+
+
+## Motivation
+
+As a (prospecting-)hoster, I want to have a quick working application for my
+main use case: library system or data management, so I can evaluate it
+right away (prospecting) and/or start using it (configuration/development). The
+RDM choice implies different kinds of data management for which I want to leverage
+pre-configuration / pre-customizations in order to get started faster and adopt
+same settings as other installations in my domain.
+
+As an InvenioRDM (core-, extension-, plugin-)developer for domain stakeholder,
+I want to be able to satisfy the exclusive needs/wants of my specialized
+stakeholders while still being able to develop and use domain authorities **within**
+the InvenioRDM context. I have a dedicated mechanism and space to insert
+customizations shared by instances for my domain this way.
+
+
+## Terminology
+
+**flavour**: A *flavour* represents the main use case of the application
```

Comment:
Yes, changing **flavour** for **product** makes sense and is clearer. We could even then have an option for `invenio` and it would just install the `cookiecutter-invenio-instance` . 

I am fine with shifting the focus on the subsequent **repository profiles**, but I'd still keep the (renamed) description for **product** around in order to give context.